settings.gradle

```diff
@@ -1,10 +1,18 @@
+pluginManagement {
+    repositories {
+        gradlePluginPortal()
+        google()
+        mavenCentral()
+    }
+}
 include 'libs:SalesforceAnalytics'
 include 'libs:SalesforceSDK'
 include 'libs:SmartStore'
 include 'libs:MobileSync'
 include 'libs:SalesforceHybrid'
 include 'libs:SalesforceReact'
 include 'native:NativeSampleApps:RestExplorer'
+include 'native:NativeSampleApps:MobileSyncCompose'
```

Comment:
Could we rename this to `MobileSyncExplorerKotlin`?

Reply:
`MobileSyncCompose` was the working title; it's no problem to change it 👍